Доработал функционал корзины полностью

Rubsee opened the pull request.

Rubsee committed:
Доработал функционал корзины полностью

captain-zsa reviewed (commented):
Ревью тут: https://disk.yandex.ru/d/0YJVU9ZMmEeUoQ

Rubsee committed:
Работа над ошибками

1. Сделал один блок(status-order) для модалок ‘error-order  и confirm-order’ и добавить модификаторы
2. Сделал обертку для minus/plus чтобы рализовать ‘|’ палочки через before и after
3. Добавил formatPrice
4. Переписал countsData через простые условия и оптимизировал
5. Заменил Drop-shadow на box-shadow
